script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -205,17 +217,18 @@ Are you that happy at seeing me again?"
 [sha:aa66e39f059e598db6f6be84eb4428b812f9d480]{
 -- Page 366, Offset 15698.
 -- あまりに能天気な発言に知性が回復した。
-The sheer upbeat audacity of her statement brings my rational thinking back to life
+The sheer upbeat audacity of her statement is enough to kick my brain back in gear.
 }
 [sha:7541340933e397d7af956c7b786e48b566010b69]{
 -- Page 366, Offset 15699.
 -- 最速で扉を閉めて鍵をロック、鞄を放り出してベッドまで一直線に突進する。
-I close, then lock my door as fast as I can, throw away my bag and make a furious beeline toward the bed.
+I close and lock the door as fast as I can, dump my bag on the floor and make a furious beeline toward the bed.
 }
 [sha:3d8402cef6d58993e9dca20c28040d6e8749e22f]{
 -- Page 367, Offset 15700. Mods: @n.
 -- 「なんど俺を驚かせれば気が済むんだオマエは!?　だいたい日も沈みきってないのに出歩くなんて、
-"Just how many times are you gonna surprise me before you're satisfied?! And hell, what are you doing out walking before the sun has se――
+"Just how many times are you gonna surprise me before you're satisfied?!
+ And what the hell are you doing out and about before the sun has se―――
```

Comment:
just reading this in PR, I was under the impression this was Arach for some reason so I was like "wtf is shiki talking about"